packages/app-mobile/components/KeyboardAvoidingView.tsx

```
@@ -14,13 +14,16 @@ const KeyboardAvoidingView: React.FC<Props> = ({ enabled, children, ...forwarded
 	return <NativeKeyboardAvoidingView
-		behavior='padding'
+		behavior={behavior}
 		{...forwardedProps}
 		enabled={enabled}
 	>
 		{children}
 	</NativeKeyboardAvoidingView>;
 };
 
-export default KeyboardAvoidingView;
+export default KeyboardAvoidingView;
```

ESLint

[error] Newline required at end of file but not found.

(eol-last)